### DISCUSSION TITLE
why the docker re-labeling? and missing docker labels

### DISCUSSION THREAD

**Discussion by tcurdt** [grafana-cold-storage/agent#2673] [Q&A]
Why is all this relabeling needed?

        - job_name: integrations/docker
          docker_sd_configs:
            - host: unix:///var/run/docker.sock
              refresh_interval: 5s
          relabel_configs:
            - action: replace
              replacement: integrations/docker
              source_labels:
                - __meta_docker_container_id
              target_label: job
            - source_labels:
                - __meta_docker_container_name
              regex: '/(.*)'
              target_label: container
            - source_labels:
                - __meta_docker_container_log_stream
              target_label: stream

And I am also still missing the labels from the container.

Which from the docs, sounds like they should be passed through by default.
Is this really correct like this?

**Comment by rfratto** [comment]
Unfortunately, the Grafana Agent team doesn't write that config you shared. It seems like that comes from Grafana Cloud; you should contact support to get more help.

**Reply by tcurdt** [reply]
That very support pointed me to ask here :)

I guess the question is why the Grafana agent sends the information in a different format that the cloud expects.
And by that every customer has to use a relabel config.

At least that's how I understand the current situation from the support conversation.
